src/sagemaker/image_uri_config/pytorch.json

```diff
@@ -54,7 +54,8 @@
             "1.3": "1.3.1",
             "1.4": "1.4.0",
             "1.5": "1.5.0",
-            "1.6": "1.6.0"
+            "1.6": "1.6.0",
+            "1.7": "1.7.0"
```

Comment:
Should this alias be to 1.7.1?

Reply:
Yes. Fixed.

Reply:
Pushed code with the fix for both inference and training version_aliases.